include/sys/pool.h

```diff
@@ -19,8 +19,12 @@ typedef struct pool pool_t;
 /*! \brief Called during kernel initialization. */
 void init_pool(void);
 
-/*! \brief Creates a pool of objects of given size. */
-pool_t *pool_create(const char *desc, size_t size);
+/* Default pool item alignment. */
+#define P_DEF_ALIGN sizeof(uint64_t)
+
+/*! \brief Creates a pool of objects of given size
```

Comment:
You can remove that `\brief`,
Actually everywhere in that file

Reply:
While this is a good idea, it's definitely isn't a good idea to do this in this PR because it would cloud the real changes.